docs/release-notes.md

```diff
@@ -0,0 +1,50 @@
+---
+page_title: Docker Swarm Release Notes
+page_description: Swarm release notes
+page_keywords: docker, swarm, clustering, discovery, release, notes
+---
+
+# Docker Swarm Release Notes 
+
+This page shows the cumulative release notes across all releases of Docker Swarm.
+
+## Version 0.3.0 (June 16, 2015)
+
+For complete information on this release, see the
+[0.3.0 Milestone project page](https://github.com/docker/swarm/wiki/0.3.0-Milestone-Project-Page).
+In addition to bug fixes and refinements, this release adds the following:
+
+- **API Compatibility**: Swarm is now 100% compatible with the Docker API. Everything that runs on top of the Docker API should work seamlessly with a Docker Swarm.
```

Comment:
The style is not the same as the 0.2.0 release notes.
Here is `** title **` and below it's `* sentence`

Is it ok ?